### ISSUE TITLE
Isn't `__proto__` standard now? Why do I get a deprecation warning?

### ISSUE THREAD

**Issue by serapath** [feross/standard#914]
Since `__proto__` became standard, i like to use it a lot and feel it makes it totally straight forward to set the prototype chain.

What is the reason that there is a warning?
Since it's standard now - would it be an option to remove the warning? :-)

**Comment by Kovensky** [comment]
It is deprecated. `__proto__` is specified in Annex B, and it is done to document what browsers happen to do in reality, even if it is not desirable.

https://tc39.github.io/ecma262/#sec-additional-ecmascript-features-for-web-browsers

"This annex describes various legacy features and other characteristics of web browser based ECMAScript implementations. All of the language features and behaviours specified in this annex have one or more undesirable characteristics and in the absence of legacy usage would be removed from this specification."

(...)

"These features are not considered part of the core ECMAScript language. Programmers should not use or assume the existence of these features and behaviours when writing new ECMAScript code. "

**Comment by serapath** [comment]
Ok, then I will just ignore the warning and continue to use it.
I find it highly desirable. It makes things very straightforward that are otherwise unnecessarily complicated.
Have never seen a good argument why `__proto__` would be a bad idea other than the plain opinion that it would somehow be undesirable. I find `class` a lot more undesirable and it's even part of the standard :/

**Comment by Aladdin-ADD** [comment]
no. it is recommended to use `Object.getPrototypeOf() ` and `Object.setPrototypeOf() ` (ES6).
